client/src/components/App.js

```diff
@@ -43,3 +46,5 @@ const App = (props) => {
 };
 
 export default hot(App);
+
+library.add(fas, far)
```

Comment:
lets add the addition of font awesome into the main.js since this is an app-wide configuration.